src/metadata/src/Command/BaseCommand.php

```diff
@@ -31,4 +33,27 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     {
         $this->io = new SymfonyStyle($input, $output);
     }
+
+    /**
+     * @throws ClientExceptionInterface when connections to Hasura have problems.
+     */
+    abstract protected function doExecute(InputInterface $input, OutputInterface $output);
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        try {
+            $result = $this->doExecute($input, $output);
+        } catch (HttpExceptionInterface $exception) {
+            $this->io->error($exception->getResponse()->getContent(false));
+            $this->io->error(self::INFO_CHECK_SERVER_CONFIG);
```

Comment:
As discussed before, are you want to keep it?

Reply:
@vuongxuongminh 

I forgot to delete it.
Thank you for your notice.